### PR TITLE
Introduce ToggleModuleStatusCommand

### LINKED ISSUE #36978: Admin API - Enable/disable module
# Tests
## Back Office
GIVEN: an enabled module
WHEN: I call the endpoint `PUT /module/status/{technicalName}` and I browse the module manager page
THEN: the module no longer appears in the disabled module list but appears in the enabled module list

GIVEN: a disabled module
WHEN: I call the endpoint `PUT /module/status/{technicalName}` and I browse the module manager page
THEN: the module no longer appears in the enabled module list but appears in the disabled module list

## API
GIVEN: an enabled module
WHEN: I call the endpoint `PUT /module/status/{technicalName}`, and I call the `GET /modules`
THEN: the status of the module has been updated with the following value : `disabled`

GIVEN: an disabled module
WHEN: I call the endpoint `PUT /module/status/{technicalName}`, and I call the `GET /modules`
THEN: the status of the module has been updated with the following value : `enabled`

GIVEN: an enabled module
WHEN: I call the endpoint `PUT /module/status/{technicalName}`, and I call the `GET module/{technicanlName}`
THEN: the status of the module has been updated with the following value : `disabled`

GIVEN: an disabled module
WHEN: I call the endpoint `PUT /module/status/{technicalName}`, and I call the `GET module/{technicanlName}`
THEN: the status of the module has been updated with the following value : `enabled`

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Introduce ToggleModuleStatusCommand CQRS command
| Type?             | new feature
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI green
| UI Tests          | Partially fixes #36978
| Fixed issue or discussion?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Comment by kpodemski** [comment]
Multi-store support to be done later or wasn't anticipated?

**Review by boherm — APPROVED** [review]

**Review by tleon — APPROVED** [review]

**Comment by jolelievre** [comment]
@kpodemski anticipated to be done later 😉
This PR was mostly an example for the MBO team that will handle the missing CQRS command for the module domain

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

File: composer.json
Comment (PrestaEdit) [inline composer.json:260]: Sure to change the related user ?

Comment (jolelievre) [reply composer.json:260]: Yep, it's temporary, as the module depends on the core and the core depends on the module (because it runs the module's tests) We have a "circular dependency" that forces us to temporarily use the module's fork branch to check that all tests are good

Then we merge the module's PR and finally we can update the core composer back to the original repository

**Pull request merged**
